torch/autograd/graph.py

```diff
@@ -132,3 +133,17 @@ def unpack_from_cpu(packed):
             return tensor.to(device, non_blocking=pin_memory)
 
         super().__init__(pack_to_cpu, unpack_from_cpu)
+
+
+@contextlib.contextmanager
+def _disable_saved_tensors_hooks(error_message):
```

Comment:
Why not make this public?

Reply:
Do you want it as a public API?

Reply:
If only functorch uses it, that's ok. But if more things starts to use it, I think we might as well make it a public API yes.